src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -22,16 +22,16 @@ public class MvcOptions
         public MvcOptions()
         {
             Conventions = new List<IApplicationModelConvention>();
-            ModelBinders = new List<ModelBinderDescriptor>();
+            ModelBinders = new List<IModelBinder>();
             ViewEngines = new List<ViewEngineDescriptor>();
```

Comment:
cc @lodejard / @davidfowl 

I left it as `ViewEngineDescriptor` on `MvcOptions`. `RazorViewEngine` requires 4 services and it feels a bit dirty to be doing `GetRequiredService` to get to those as part of execution. Thoughts?